### ISSUE TITLE
Bookmarks

### ISSUE THREAD

**Issue by fkuehne** [pdherbemont/Glasses#53]
It would be nice to have a working and savable boobmarks feature in Lunettes the same way it should have been implemented in VLC for a long time.

**Comment by CarterA** [comment]
Would bookmarks be stored in the media file as metadata, or in the app preference file?

**Comment by fkuehne** [comment]
Probably in the app preference data (within a CoreData facility) as this is a lot easier and allows support for all kinds of file-formats supported by VLC. A bunch of container formats supported by VLC don't allow bookmarks (such as AVI, ASF, MPEG TS, ...). Therefore, this approach would make sense IMO.

**Comment by pdherbemont** [comment]
Yes, the CoreData DB is a good idea for that IMO. It would need to be extended, but it's not a big deal.
